blocks/core/ff_module/ff_module-online-status/ff_module-online-status.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var React = require('react');
+
+module.exports = React.createClass({
+    displayName: 'OnlineStatus',
+    propTypes: {
+        state: React.PropTypes.shape({
+            offline: React.PropTypes.bool,
+            online: React.PropTypes.bool
```

Comment:
I'd just have one of these, otherwise you leave the door open for weird states where the component is both online and offline.